chapter-03/recipe-04/cxx-example/linear-algebra.cpp

```diff
@@ -4,11 +4,8 @@
 #include "CxxBLAS.hpp"
 #include "CxxLAPACK.hpp"
 
-#include <chrono>
-#include <cstdlib>
 #include <iostream>
 #include <random>
-#include <vector>
```

Comment:
Why remove the `#include <vector>`? It's still used in the code.

Reply:
OK I will check, I was possibly too quick.